_shared/project/Makefile

```diff
@@ -34,7 +36,9 @@ web: python
 .PHONY: shell
 $(call help,make shell,"launch a Python shell in this project's virtualenv")
 shell: python
-{% if cookiecutter._directory == 'pyramid-app' %}
+{% if cookiecutter._directory == 'pyapp' %}
+	@pyenv exec tox -qe dev --run-command 'ipython'
```

Comment:
There's no `pshell` for non-web apps to run so for non-web apps `make shell` just runs `ipython` directly. (`pshell` is a Pyramid thing, it adds some handy Pyramid integration into IPython.)